tests/tests_unit/test_utils/test_time.py

```diff
@@ -36,6 +38,44 @@
     import pandas
 
 
+@pytest.mark.parametrize(
+    "offset_inp, expected",
+    (
+        ("", timedelta(0)),
+        ("01:15", timedelta(seconds=4500)),
+        ("01:15:12", timedelta(seconds=4512)),
+        ("23:59", timedelta(seconds=86340)),
```

Comment:
We have disallowed (400 error) fixed offsets that are not multiple of 15 minutes. Not sure if this matters to you.

Reply:
We parse it and just pass it on to the API (and let it decide/raise) - this way the SDK is future-proof in case you ever decide to be more lenient with offsets 😄